src/graphics/aurora/model_kotor.cpp

```diff
@@ -557,7 +555,28 @@ void ModelNode_KotOR::readOrientationController(uint8 columnCount, uint16 rowCou
 		uint16 timeIndex, uint16 dataIndex, std::vector<float> &data) {
 	switch (columnCount) {
 		case 2:
-			// TODO: 2 column orientation controller
+			for (int r = 0; r < rowCount; r++) {
+				QuaternionKeyFrame q;
+				q.time = data[timeIndex + r];
+
+				uint32 temp = *reinterpret_cast<uint32 *>(&data[dataIndex + r]);
```

Comment:
Yeah, that's not really portable, unfortunately. Potentially, the data might be in a different endianness (float endianness can differ from integer endianness) and even a different format than IEEE (*), so we can't necessarily guarantee that it's the same bit-pattern anymore.

You will probably need to rewrite the loading of the controller data altogether. Read in a byte array, and read the correct data from that here (using the READ_LE_UINT32() etc. macros and convertIEEEFloat()).

(*) If we supported that in readIEEEFloatLE(), which we don't yet, yes